specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2022-03-01-preview/streamingjobs.json

```diff
@@ -0,0 +1,1126 @@
+      "properties": {
+        "value": {
+          "description": "List of Stream Analytics operations supported by the Microsoft.StreamAnalytics resource provider.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Operation": {
+      "description": "A Stream Analytics REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the operation being performed on this particular object.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
+        },
+        "display": {
+          "type": "object",
+          "description": "Contains the localized display information for this particular operation / action.",
+          "readOnly": true,
+          "properties": {
+            "provider": {
+              "description": "The localized friendly form of the resource provider name.",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "The localized friendly form of the resource type related to this action/operation.",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "The localized friendly name for the operation.",
+              "type": "string",
+              "readOnly": true
+            },
+            "description": {
+              "description": "The localized friendly description for the operation.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "StreamingJob": {
+      "type": "object",
+      "description": "A streaming job object, containing all information associated with the named streaming job.",
+      "allOf": [
+        {
+          "$ref": "../../common/v1/definitions.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/StreamingJobProperties",
+          "description": "The properties that are associated with a streaming job.  Required on PUT (CreateOrReplace) requests.",
+          "x-ms-client-flatten": true
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Describes the managed identity assigned to this job that can be used to authenticate with inputs and outputs."
+        }
+      }
+    },
+    "StreamingJobProperties": {
```

Comment:
>StreamingJobProperties [](http://example.com/codeflow?start=5&length=22)

Removing the transformation property from this object is a cross-version breaking change. You will need to complete breaking change review unless all previous API versions with the transformation property are still hidden.